packages/affine/shared/src/services/doc-mode-service.ts

```diff
@@ -9,13 +9,15 @@ import { type Disposable, Slot } from '@blocksuite/global/utils';
 const DEFAULT_MODE = DocMode.Page;
 
 export interface DocModeProvider {
-  setMode: (mode: DocMode, docId?: string) => void;
-  getMode: (docId?: string) => DocMode;
-  toggleMode: (docId?: string) => DocMode;
-  onModeChange: (
+  setPrimaryMode: (mode: DocMode, docId?: string) => void;
```

Comment:
Let's also remove the `DocMode` enum and use pure string literal type (`page` | `edgeless`) instead.